app/mailers/admin_mailer.rb

```diff
@@ -1,6 +1,6 @@
 class AdminMailer < ActionMailer::Base
   add_template_helper(ApplicationHelper)
-  default from: 'no-reply@reservations.app'
+  default from: "no-reply@#{ ActionMailer::Base.default_url_options[:host] }"
```

Comment:
Would this result in e.g. no-reply@reservations.yale.edu ?

But yes. Thanks! :grinning: